### PR TITLE
Change view ranking filter header

### LINKED ISSUE #6201: Add a clear distinction for filter headers
image

Currently the filter headers `Variant` and `Show` blend into other filter options and it's not obvious which one is not the option.

### PR DESCRIPTION
**Pull request by gagahpangeran**
Closed #6201 

I only change based on the design and don't modify `Sort` component to prevent changes in other page.

**Desktop**

image

**Mobile**

image

The mobile view is cover up half of the screen. Should I make it horizontal?

### CODE REVIEW

**Comment by notbakaneko** [comment]
I think the vertical padding can be reduced all around and the title text should be a bit smaller.
Title when in mobile view should probably be inline

**Comment by gagahpangeran** [comment]
image

image

File: resources/assets/less/bem/ranking-filter.less [hunk 1]
```diff
@@ -13,17 +14,38 @@
...
+      display: block;
+
       @media @desktop {
         flex: 1;
       }
+
+      .@{top}__item--title {
+        padding: 0;
+        margin-bottom: 10px;
+      }
```
Comment (notbakaneko) [inline resources/assets/less/bem/ranking-filter.less:48]: This should be inside `&--title` as `.@{top}__item--full &` instead.

**Comment by gagahpangeran** [comment]
Feels empty

image

I can't check spotlight page yet because the spotlight seeder is error.

**Comment by gagahpangeran** [comment]
All done! Please check again.

**Pull request merged**